packages/ra-core/src/util/hooks.ts

```diff
@@ -73,6 +73,7 @@ export function useTimeout(ms = 0, key = '') {
 export function useIsMounted() {
     const isMounted = useRef(true);
```

Comment:
Shouldn't be initialized with `false` ?

Reply:
No, as it would trigger a useless re-render